src/main/java/org/jenkinsci/test/acceptance/utils/pluginTests/SecurityDisabler.java

```diff
@@ -47,7 +50,11 @@ public void stopUsingSecurityAndSave() {
         Control use = control("/useSecurity");
         if(use.resolve().isSelected()) {
             use.click();
-            control("/Submit").click();
+            WebElement submit = control("/Submit").resolve();
+            // Would be better to use Control.clickAndWaitToBecomeStale,
+            // but it's not visible because it's still experimental
+            submit.click();
+            waitFor(submit).until(Control::isStale);
```

Comment:
can this abuse 'submit'?

Reply:
The same. This is not a submit type  button